### PR TITLE
Some minor change which are required to make this work on other systems

### PR DESCRIPTION
**Pull request by heistermann**
See commits.

### CODE REVIEW

File: wradvis/glcanvas.py [hunk 1]
```diff
@@ -99,6 +99,7 @@ def __init__(self, **kwargs):
                            method='subdivide',
                            #interpolation='bicubic',
                            cmap=cmap,
+                           clim=(0,10),
```
Comment (kmuehlbauer) [inline wradvis/glcanvas.py]: clim=(0, 10) ?

Then I get an almost black image. I'm reading RX in my case. Maybe we should agree on some source dataset? What do you suggest?

**Comment by kmuehlbauer** [comment]
One minor comment! Otherwise LGTM!

**Comment by heistermann** [comment]
Sorry, setting clim was just a quick fix because my images were all white.

RX is dBZ, right? I was looking at hoourly depths (RW). Would clim=(0,60) be a compromise that suits both mm/h and dBZ as a default?

Having an example data set sounds like a good idea. This should also be the initial directory. I don't have a strong opinion which data to use, though. Maybe better to have something that is already in mm/h. Some RY data for an event?

**Comment by kmuehlbauer** [comment]
We could take one day of RW-data. Not only because it is "freely" available. Then we can have hardcoded clims to begin with and do not have to bother with different limits.

**Comment by heistermann** [comment]
I'll put something in,

**Comment by kmuehlbauer** [comment]
Shall we merge this anyway?

**Comment by heistermann** [comment]
I added RW example data. Should I merge now?

**Comment by kmuehlbauer** [comment]
Done!

**Pull request merged**